.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+.idea
```

Comment:
I think `.idea` probably can be moved to your own global .gitignore.

I agree `node_modules` should definitely be in here though.